assets/app/view/game/round/stock.rb

```diff
@@ -369,7 +369,7 @@ def render_buy_input_interval(company)
         end
 
         def render_bid_input(company)
-          return [] unless @step.can_bid?(@current_entity, company)
+          return [] unless @step.respond_to?(:can_bid_company?) && @step.can_bid_company?(@current_entity, company)
```

Comment:
why was this changed?

Reply:
this looks like a very different change, why not continue to call it can_bid?

Reply:
why use respond_to?

Reply:
18NY hit view code that was originally was only hit by 1822 and used methods that only exist in the 1822 code base. The 1822 can_bid? method collided with the 18NY can_bid? method, which had a different signature and caused issues. This view code assumed that if you can 'buy_company' and 'bid', then you must be able to bid on companies (bad assumption and needed to be fixed).

Upon investigation into the code base, the rest of the code base matched the 18NY can_bid? signature, so I left that one alone and instead changed the 1822 specific can_bid? to can_bid_company?, which is what the method is really trying to determine (vs bidding to start a corporation).  Since only 1822 uses that method, I included respond_to?, so that other games weren't required to implement the method unless they needed it. This is standard practice throughout the view code for game specific behavior.